mobilidade_rio/mobilidade_rio/pontos/views.py

```diff
@@ -189,6 +189,14 @@ def get_queryset(self):
         # increase performance if no need to raw query
         raw_filter_used = False
 
+        # stop_code - parent_station or itself
```

Comment:
boa! em relacao ao filtro usado em `stop_id__all`, esse tipo de filtragem é mais otimizado? da para padronizarmos?

Reply:
e pq um tem `__all` e outro não? 🤔

Reply:
@fernandascovino 
* `stop_id_all` obtém stops que tenham trips em comum, dado uma lista de stops como input
* `stop_id`
  - Se `stop_id` for de um pai, filtra por stops filho desse pai
  - Se `stop_id` for de um filho, filtra por stop_id desse filho